roxie/ccd/ccdcontext.cpp

```diff
@@ -1671,6 +1673,7 @@ class CRoxieContextBase : implements IRoxieSlaveContext, implements ICodeContext
     }
     virtual IEngineContext *queryEngineContext() { return NULL; }
     virtual char *getDaliServers() { throwUnexpected(); }
+    virtual unsigned getWorkflowId() { return 0; } // this is a virtual which is implemented in IGlobalContext
```

Comment:
This was the simplest way to get at the information.  I suspect that beginGraph/endGraph should actually be implemented in the derived context that also defines this function.  Rather than investigate moving them, this was less intrusive.